### DISCUSSION TITLE
proje4 postorder çıktısı

### DISCUSSION THREAD

**Discussion by zeynepssasmaz** [gusanmaz/NKU_DS_Course_2022#15] [Q&A] [answered]
proje4te postorder çıktısının da hatalı olduğunu düşünüyorum. Çıktının  -100  4  12  50  75  100  45  9 olması gerekiyor.

**Comment by gusanmaz** [comment, marked as answer]
Evet dogrusu  `-100 4 12 50 75 100 45 9` olacak. Tesekkurler.
